VAMobile/src/screens/HealthScreen/Vaccines/NoVaccineRecords/NoVaccineRecords.test.tsx

```diff
@@ -1,36 +1,34 @@
 import 'react-native'
 import React from 'react'
+import { screen, fireEvent } from '@testing-library/react-native'
 
-import { context, render, RenderAPI } from 'testUtils'
-import { act } from 'react-test-renderer'
-
+import { context, render } from 'testUtils'
 import NoVaccineRecords from './NoVaccineRecords'
-import { Linking, TouchableWithoutFeedback } from 'react-native'
+import { Linking } from 'react-native'
```

Comment:
This can be grouped with the imports above

Reply:
updated